reactor-core/src/test/java/reactor/guide/GuideTests.java

```diff
@@ -201,21 +201,20 @@ public void advancedCold() {
 
 	@Test
 	public void advancedHot() {
-		Sinks.StandaloneFluxSink<String> hotSource = Sinks.multicastNoWarmup();
-
-		Flux<String> hotFlux = hotSource.asFlux().map(String::toUpperCase);
+		Sink<String> hotSource = Sinks.replay().limit(0);
```

Comment:
reiterating that `replay().limit(0)` is probably a bit too confusing. `multicast().onBackpressureError()` sounds like the best example to give, showcasing only live data makes it past the sink.

Also change in `advancedFeatures.adoc`.

Reply:
fixed in 7b6b655